README.md

```diff
@@ -19,28 +19,32 @@ Working on getting an Apple Development account for official distribution...
 
 ---
 
-I think the idea of recording everything you see has the potential to change how we interact 
+I think the idea of recording everything you see has the potential to change how we interact
 with our computers, and believe it should be open source.
 
-Also, from a privacy / security perspective, this is like... pretty scary stuff, and I want the code open 
-so we know for certain that nothing is leaving your laptop. Even logging to Sentry has the potential to 
+Also, from a privacy / security perspective, this is like... pretty scary stuff, and I want the code open
+so we know for certain that nothing is leaving your laptop. Even logging to Sentry has the potential to
 leak private info.
 
 This is 100% local. Please, read the code yourself.
 
 ### This is crazy alpha version
+
 I wrote this in a couple days over the holidays, and if there's one takeaway, it's that I'm a
 complete novice at Swift.
 
 ## Build it yourself
+
+- Clone the repo `git clone --recursive -j8 https://github.com/jasonjmcghee/rem.git`
 - Open project in Xcode
-- Clone `https://github.com/stephencelis/SQLite.swift` and add it to the project
+- Change default SQLite.Swift sdk archiecture to macOS <img width="1512" alt="Screenshot 2023-12-28 at 5 38 19 PM" src="https://github.com/ruslanjabari/rem/assets/59275080/63c08975-0bd2-4fe8-91ca-0b9406d44704">
```

Comment:
Can we fix this by setting it here? Or is this just built into `SQLite.swift`?

<img width="1405" alt="image" src="https://github.com/jasonjmcghee/rem/assets/1522149/6400bcea-7833-4eb3-bb3d-d9c9ec0427bd">

Reply:
good question. I was running into https://github.com/jasonjmcghee/rem/issues/10#issue-2059025888 when I was trying to do it that way so I just gave up and changed the architecture the way I pasted it here - wasn't able to make it build otherwise.